samples/bootloader/README.rst

```diff
@@ -86,37 +70,39 @@ The sample accomplishes a simple and reliable *Root of Trust* (RoT) for a secure
 Provisioning
 ============
 
-The public key hashes are not compiled with the source code of the bootloader sample.
-Instead, they must be stored in a separate memory region through a process called *provisioning*.
+The public key hashes are not compiled with the source code of the NSIB.
+Instead, they must be written to the device in a process called *provisioning*.
+
+The hashes are automatically generated by the build system based on the specified private key and the additional public keys.
 
-By default, the bootloader sample automatically generates and provisions public key hashes directly into the bootloader HEX file, based on the specified private key and the additional public keys.
+By default, the hashes are placed directly into the NSIB HEX file and then automatically provisioned when the HEX file is programmed to the device.
 
-Alternatively, to facilitate the manufacturing process of a device using the bootloader sample, it is possible to decouple this process and program the sample HEX file and the HEX file containing the public key hashes separately.
-If you choose to do so, use the Python scripts located in the :file:`scripts/bootloader` folder to create and provision the keys manually.
+Alternatively, in a more realistic manufacturing process, the NSIB HEX file and the HEX file containing the hashes can be programmed separately.
+For this approach, use the Python scripts located in the :file:`scripts/bootloader` folder.
 
-In both cases, the bootloader accesses the provisioned data using the :ref:`doc_bl_storage` library.
+In either case, the NSIB accesses the provisioned data at runtime using the :ref:`doc_bl_storage` library.
 
 .. _bootloader_provisioning_otp:
 
-OTP Regions
+OTP regions
 -----------
 
-On some SoCs/SiPs, such as the nRF9160 or nRF5340, the provisioned data is held in the *one-time programmable* (OTP) region in the *User Information Configuration Registers* (UICR).
+For the nRF SoCs that support it, such as the nRF9160 or nRF5340, the provisioned data is held in the *one-time programmable* (OTP) region in the *User Information Configuration Registers* (UICR) instead of the internal flash memory.
 OTP regions are special regions of the flash memory that only allow flash memory writes in half-word lengths, and *only* when the target half-word is the value ``0xFFFF``.
```

Comment:
```suggestion
One-time programmable (OTP) regions are special regions of the *User Information Configuration Registers* (UICR) that only allow flash memory writes in half-word lengths, and *only* when the target half-word is the value ``0xFFFF``.

On the SoCs that support OTP regions, such as the nRF9160 and nRF5340, the provisioned data is held in the OTP regions instead of the internal flash memory.
```

Reply:
@SebastianBoe Do OTP regions allow memory writes only when the target half-word is 0xFFFF or only when it is not?
Also, are there more then one OTP region in the supported SoCs?

Reply:
New phrasing is a bit ambiguous to how many OTP regions there are in the UICR, don't you agree?

Reply:
Updated (that's why I asked)